internal/context/context.go

```diff
@@ -91,7 +92,7 @@ func (s *SMFContext) ListenIP() net.IP {
 // RetrieveDnnInformation gets the corresponding dnn info from S-NSSAI and DNN
 func RetrieveDnnInformation(Snssai *models.Snssai, dnn string) *SnssaiSmfDnnInfo {
 	for _, snssaiInfo := range GetSelf().SnssaiInfos {
-		if snssaiInfo.Snssai.Sst == Snssai.Sst && snssaiInfo.Snssai.Sd == Snssai.Sd {
+		if snssaiInfo.Snssai.Sst == Snssai.Sst && strings.EqualFold(snssaiInfo.Snssai.Sd, Snssai.Sd) {
```

Comment:
modify to `if openapi.SnssaiEqualFold(snssaiInfo, Snssai) {`

Reply:
```
type SnssaiSmfInfo struct {
	Snssai   SNssai
	DnnInfos map[string]*SnssaiSmfDnnInfo
}
```
The `snssai` in `snssaiInfo` is not `model.snssai`, but `SNssai` defined in internal/context/snssai.go. 
`snssaiEqualFold` uses `model.snssai`. 
Should I make the conversion (from `context.snssai` to `model.snssai`) in order to use `snssaiEqualFold`?

Reply:
modify to
```
if snssaiInfo.Snssai.EqualModelsSnssai(Snssai) {
```